crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -62,6 +62,7 @@ pub fn generate_cpp_properties(
 pub mod tests {
```

Comment:
entry in the CHANGELOG would be good to under Added for adding cxx_name / rust_name support to qproperty attribute